test/integration/globals/default-route.js

```diff
@@ -68,15 +68,11 @@ context('patient page', function() {
 
   specify('current clinician has no team', function() {
     cy
-      .routeGroupsBootstrap(_.identity, null, fx => {
-        const currentClinician = _.find(fx.data, clinician => clinician.id === '11111');
-
-        currentClinician.relationships.team = { data: null };
-
-        return fx;
-      })
```

Comment:
I think `routeGroupsBootstrap()` and `routeCurrentClinician()` are doing the same thing?

Test passes with `routeGroupsBootstrap()` removed.

Reply:
Ah that probably is true now